colab/nessie-iceberg-demo-nba.ipynb

```diff
@@ -2020,7 +2020,7 @@
     }
    ],
    "source": [
-    "spark.sql(\"select count(*) from nessie.nba.salaries\").toPandas()"
+    "spark.sql(\"select count(*) from nessie.nba.salaries\").show()"
```

Comment:
For a nicer representation, I'd prefer `toPandas`, but that has issues on Colab. Think, we need to figure out how to fix that (not at the moment, but later)